runtime-config-linux.md

```diff
@@ -152,12 +160,38 @@ For example, to run a new process in an existing container without updating limi
 
 #### Disable out-of-memory killer
 
+`disableOOMKiller` contains a flag (`true` or `false`) that enables or disables
+the Out of Memory killer for a cgroup. If enabled (`false`), tasks that attempt
+to consume more memory than they are allowed are immediately killed by the OOM killer.
```

Comment:
And since we have a default (`false`), I'd suggest leading in with the usual pattern:

> **`disableOOMKiller`** (bool, optional) contains…

Or whatever we decide our pattern for introducing JSON fields is going to be.